mlflow/deployments/server/app.py

```diff
@@ -204,8 +220,12 @@ def create_app_from_config(config: GatewayConfig) -> GatewayAPI:
     """
     Create the GatewayAPI app from the gateway configuration.
     """
+    limiter = Limiter(
```

Comment:
we can easily add root level config by using the `default_limits` argument of Limiter
https://slowapi.readthedocs.io/en/latest/api/

Reply:
Thanks for the investigation! Would `default_limits` help simplify the code?

Reply:
No. It can be used when we want to enable users to have root-level config. Since root level config is out of the scope of this ticket (Decision #5 in the design doc), we can skip it.

Reply:
Got it. Yes, the root level config is out of the scope.